wpilibc/src/generate/main/native/include/frc/hid.h.jinja

```diff
@@ -0,0 +1,202 @@
+   * is bound to the range of [0, 1] as opposed to the usual [-1, 1].
+   *
+   * @return the axis value.
+   */
+  double Get{{ trigger.HumanName }}Axis() const;
+{% if trigger.UseThresholdMethods %}
+  /**
+   * Constructs an event instance around the axis value of the {{ trigger.DocName }}.
+   * The returned trigger will be true when the axis value is greater than
+   * {@code threshold}.
+   * @param threshold the minimum axis value for the returned event to be true.
+   * This value should be in the range [0, 1] where 0 is the unpressed state of
+   * the axis.
+   * @param loop the event loop instance to attach the event to.
+   * @return an event instance that is true when the {{ trigger.DocName }}'s axis
+   * exceeds the provided threshold, attached to the given event loop
+   */
+  BooleanEvent {{ trigger.HumanName }}(double threshold, EventLoop* loop) const;
+
+  /**
+   * Constructs an event instance around the axis value of the {{ trigger.DocName }}.
+   * The returned trigger will be true when the axis value is greater than 0.5.
+   * @param loop the event loop instance to attach the event to.
+   * @return an event instance that is true when the {{ trigger.DocName }}'s axis
+   * exceeds 0.5, attached to the given event loop
+   */
+  BooleanEvent {{ trigger.HumanName }}(EventLoop* loop) const;
+{% endif -%}
+{% endfor -%}
+{% for button in buttons %}
+  /**
+   * Read the value of the {{ button.DocName|default(button.NameParts|map("lower")|join(" ")) }} button on the controller.
+   *
+   * @return The state of the button.
+   */
+  bool Get{{ button.NameParts|map("capitalize")|join }}Button() const;
+
+  /**
+   * Whether the {{ button.DocName|default(button.NameParts|map("lower")|join(" ")) }} button was pressed since the last check.
+   *
+   * @return Whether the button was pressed since the last check.
+   */
+  bool Get{{ button.NameParts|map("capitalize")|join }}ButtonPressed();
+
+  /**
+   * Whether the {{ button.DocName|default(button.NameParts|map("lower")|join(" ")) }} button was released since the last check.
+   *
+   * @return Whether the button was released since the last check.
+   */
+  bool Get{{ button.NameParts|map("capitalize")|join }}ButtonReleased();
+
+  /**
+   * Constructs an event instance around the {{ button.DocName|default(button.NameParts|map("lower")|join(" ")) }} button's
+   * digital signal.
+   *
+   * @param loop the event loop instance to attach the event to.
+   * @return an event instance representing the {{ button.DocName|default(button.NameParts|map("lower")|join(" ")) }} button's
+   * digital signal attached to the given loop.
+   */
+  BooleanEvent {{ button.NameParts|map("capitalize")|join }}(EventLoop* loop) const;
+{% endfor %}
+{% if ConsoleName == "Xbox" or ConsoleName == "Stadia" %}
+  /**
+   * Read the value of the left bumper (LB) button on the controller.
+   *
+   * @return the state of the button
+   */
+  [[deprecated("Use GetLeftBumper instead")]]
```

Comment:
Shouldn't this say `GetLeftBumperButton`? (Similar applies for `GetRightBumper`, `GetLeftBumperPressed`, `GetRightBumperPressed`, `GetLeftBumperReleased`, and `GetRightBumperReleased`)